src/test/data/JsonSerializableTagTreeTest/sampleValidTagTree.json

```diff
@@ -1,5 +1,6 @@
 {
   "tagSubTagMap" : {
+    "architecture" : [ "architecturemod" ],
```

Comment:
Pretty daring to change the sample test tree. But I guess maybe the original one might not have been comprehensive enough.

Reply:
haha, i ran the tests again to ensure nothing else was affected.